### PR TITLE
move generator require to the top

### LINKED ISSUE #904: generator require is not at the top
Over in ['app.js' file, I noticed that generator require was not done at the top](https://github.com/fossasia/open-event-webapp/blob/development/src/app.js#L59). I'd like to update it . Is this a welcomed contribution right now?

### PR DESCRIPTION
**Pull request by arrizalamin**
fixes #904

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const compression = require('compression');
 const bodyParser = require('body-parser');
 const path = require('path');
 var siofu = require("socketio-file-upload");
+var generator = require('./backend/generator.js');
```
Comment (jayvdb) [inline src/app.js]: this is http://eslint.org/docs/rules/no-undef ?  maybe eslint needs to be configured..?

Comment (arrizalamin) [inline src/app.js]: yes, I think the [node environment](http://eslint.org/docs/user-guide/configuring) should be set to true.

Comment (jayvdb) [inline src/app.js]: Do it as part of this patch, or as a separate PR?

Comment (mariobehling) [inline src/app.js]: Patch it here if possible. Cheers.

File: .eslintrc [hunk 2]
```diff
@@ -2,7 +2,8 @@
     "rules": {},
     "env": {
         "es6": true,
-        "browser": true
+        "browser": true,
+	"node": true
```
Comment (jayvdb) [inline .eslintrc]: different indentation here.

**Comment by jayvdb** [comment]
And please amend your current commit.
i.e. submit either one or two commits, not a third one to fix your own indentation bug.

**Pull request merged**